homeworks/irina.vlasova_IrinaBondarenko/homework_5/sumArray.js

```diff
@@ -0,0 +1,18 @@
+function foo() {
+    let sum = 0;
+    return (element) => {
+        sum += element;
+        return sum;
+    };
+}
+function bar(arr) {
+    const newArr = [];
+    const baz = foo();
+    for (let i = 0; i < arr.length; i++) {
+        newArr[i] = baz(arr[i]);
+    }
+    return newArr;
+}
```

Comment:
you have a proper solution, but please use `map()` method here, it will make your code much more readable